### PR TITLE
interface: group feed no longer shows in omnibox

### PR DESCRIPTION
**Pull request by tacryt-socryp**
Fixes https://github.com/urbit/landscape/issues/721

### CODE REVIEW

File: pkg/interface/src/logic/lib/omnibox.js [hunk 1]
```diff
@@ -99,9 +99,11 @@ export default function index(contacts, associations, apps, currentGroup, groups
   Object.keys(associations).filter((e) => {
     // skip apps with no metadata
     return Object.keys(associations[e]).length > 0;
-    }).map((e) => {
-      // iterate through each app's metadata object
-      Object.keys(associations[e]).map((association) => {
+  }).map((e) => {
+    // iterate through each app's metadata object
+    Object.keys(associations[e])
+      .filter((association) => !associations[e][association].metadata.hidden)
```
Comment (matildepark) [inline pkg/interface/src/logic/lib/omnibox.js:105]: This should probably be a bit safer I think, it's quite nested

Comment (tacryt-socryp) [reply pkg/interface/src/logic/lib/omnibox.js:105]: Accessing using the same pattern found in the following map? Not sure what else to do.

**Comment by matildepark** [comment]
poke @tacryt-socryp

**Comment by tacryt-socryp** [comment]
Totally forgot about this, will do

**Comment by tacryt-socryp** [comment]
okey doke @matildepark

**Review by matildepark — APPROVED** [review]

**Pull request merged**